lib/iris/tests/unit/fileformats/netcdf/load_cube/load_cube__activate/__init__.py

```diff
@@ -0,0 +1,244 @@
+        # Destroy a temp directory for temp files.
+        shutil.rmtree(cls.temp_dirpath)
+
+    def load_cube_from_cdl(self, cdl_string, cdl_path, nc_path):
+        """
+        Load the 'phenom' data variable in a CDL testcase, as a cube.
+
+        Using ncgen, CFReader and the _load_cube call.
+        Can use a genuine Pyke engine, or the actions mimic engine,
+        selected by `self.use_pyke`.
+
+        """
+        # Write the CDL to a file.
+        with open(cdl_path, "w") as f_out:
+            f_out.write(cdl_string)
+
+        # Create a netCDF file from the CDL file.
+        command = "ncgen -o {} {}".format(nc_path, cdl_path)
+        subprocess.check_call(command, shell=True)
+
+        # Simulate the inner part of the file reading process.
+        cf = CFReader(nc_path)
+        # Grab a data variable : FOR NOW, should be only 1
+        cf_var = list(cf.cf_group.data_variables.values())[0]
+        cf_var = cf.cf_group.data_variables["phenom"]
+
+        do_pyke = self.use_pyke or self.compare_pyke_nonpyke
+        do_nonpyke = not self.use_pyke or self.compare_pyke_nonpyke
+        if do_pyke:
+            pyke_engine = iris.fileformats.netcdf._pyke_kb_engine_real()
+        if do_nonpyke:
+            nonpyke_engine = iris.fileformats._nc_load_rules.engine.Engine()
+
+        iris.fileformats.netcdf.DEBUG = self.debug
+
+        # Call the main translation function to load a single cube.
+        def load_single_cube(engine):
+            # _load_cube establishes per-cube facts, activates rules and
+            # produces an actual cube.
+            cube = _load_cube(engine, cf, cf_var, nc_path)
+
+            # Also Record, on the cubes, which hybrid coord elements were identified
+            # by the rules operation.
+            # Unlike the other translations, _load_cube does *not* convert this
+            # information into actual cube elements.  That is instead done by
+            # `iris.fileformats.netcdf._load_aux_factory`.
+            # For rules testing, it is anyway more convenient to deal with the raw
+            # data, as each factory type has different validity requirements to
+            # build it, and none of that is relevant to the rules operation.
+            cube._formula_type_name = engine.requires.get("formula_type")
+            cube._formula_terms_byname = engine.requires.get("formula_terms")
+
+            return cube
+
+        if do_pyke:
+            pyke_cube = load_single_cube(pyke_engine)
+        if do_nonpyke:
+            nonpyke_cube = load_single_cube(nonpyke_engine)
+
+        # If requested, directly compare the pyke and non-pyke outputs.
+        if self.compare_pyke_nonpyke:
+            # Compare the loaded cubes from both engines.
+            # print("\nPYKE-NONPYKE COMPARE")
+
+            # First zap cube-data, as masked data does not compare well.
+            def unmask_cube(cube):
+                # preserve the original, we're going to realise..
```

Comment:
This comment could be clearer, I can't make out what "we're going to realise" is refering to.
Also, capitalise the first letter for consistency.